### ISSUE TITLE
Switch to bootstrapping for MBAR errors.

### ISSUE THREAD

**Issue by IAlibay** [OpenFreeEnergy/openfe#1012]

**Comment by IAlibay** [comment]
This probably mostly needs updating `_get_free_energy` here: https://github.com/OpenFreeEnergy/openfe/blob/main/openfe/protocols/openmm_utils/multistate_analysis.py and then checking just how slow it is.

@jthorton since you did this for the industry benchmark scripts, would you be interested in doing this at some point?

**Comment by jthorton** [comment]
Yeah, I am happy to do this, do we want to go for the full 1000 iterations? I noticed in testing for the industry stuff it takes around a minute to run which should not be noticeable overall!

**Comment by jameseastwood** [comment]
This is blocked by perses @mikemhenry
